### ISSUE TITLE
Library doesn't work in node 14, but no engine is set

### ISSUE THREAD

**Issue by aljones15** [digitalbazaar/bedrock-web-authn-token#22]
```js
Welcome to Node.js v14.19.3.
Type ".help" for more information.
> import('./lib/index.js').then(console.log).catch(console.error)
Promise {
  <pending>,
  [Symbol(async_id_symbol)]: 324,
  [Symbol(trigger_async_id_symbol)]: 323,
  [Symbol(destroyed)]: { destroyed: false }
}
> TypeError: Cannot read property 'randomFill' of undefined
    at _getRandomValues (file:///home/liminal18/Programs/nodejs/bedrock/bedrock-web-authn-token/lib/pbkdf2.js:125:13)
    at file:///home/liminal18/Programs/nodejs/bedrock/bedrock-web-authn-token/lib/pbkdf2.js:12:25
    at ModuleJob.run (internal/modules/esm/module_job.js:183:25)
    at async Loader.import (internal/modules/esm/loader.js:178:24)
    at async importModuleDynamicallyWrapper (internal/vm/module.js:451:15)
```

**Comment by dlongley** [comment]
#24 should make the library work in node 14.

**Comment by dlongley** [comment]
Fixed by #24.
